test/views/UploadResultViewSpec.scala

```diff
@@ -125,17 +125,17 @@ class UploadResultViewSpec extends UnitSpec with RasTestHelper {
 
 		"contain a cy message when upload date is 31/12/2018" in {
```

Comment:
This isn't something for this pr but something i've seen in PBIK as well, it might just be me but when i see "cy" my mind thinks welsh content. It's not abundantly clear in my mind "cy" relates to current year and in my opinion should be more explicitly annotated as such.

Reply:
https://github.com/hmrc/service-manager-config/pull/3468